ac/ac-autocode/src/ActivityRunner/TestPanel.jsx

```diff
@@ -0,0 +1,97 @@
+// @flow
+
+import React from 'react';
+import { withState } from 'recompose';
+import styled from 'styled-components';
+
+import Test from './Test';
+
+const renderLines = (text: string) =>
+  text &&
+  text
+    .split('\n')
+    .filter(x => x)
+    .map((x, index) => (
+      <div key={x} index={index}>
+        {x}
+      </div>
+    ));
+
+const CodeBox = styled.div`
+  min-width: 300px;
+  max-width: 400px;
+  border: solid 1px;
+  font-family: monospace;
+  padding: 5px;
+  margin-bottom: 15px;
+`;
+
+const ButtonList = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  margin-left: 10px;
+  margin-right: 10px;
+`;
+
+const Feedback = ({ input, expected, received, error, stdout, debug }) => (
+  <div>
+    {[
+      input && <div key="inputT">INPUT</div>,
+      input && <CodeBox key="inputB">{renderLines(input)}</CodeBox>,
```

Comment:
The input looks weird on the three examples:
```
a=[10, 2, 15, 8, 7]
print b
```
or 
```
print roman_num(50)
```
I think I'd just like the input shown to the students to be of the form 
```
a=[10, 2, 15, 8, 7]
```
or
```
num=50
```
So it could be only the preCode, then if you are afraid of students changing the variable that you created in the preCode for the postCode, we could just set it again in the post code, or I don't mind having:
```
num=50
```
then
```
print roman_num(50)
```